code/modules/projectiles/guns/minigun.dm

```diff
@@ -0,0 +1,126 @@
+// Minigun, fed via a backpack
+/obj/item/gun/minigun
+	name = "minigun"
+	desc = "A backpack-fed minigun"
+	icon = 'icons/obj/guns/plasma/hydrogen.dmi'
+	icon_state = "plasma"
+	origin_tech = list(TECH_COMBAT = 10)
+	w_class = ITEM_SIZE_BULKY
+	recoil_buildup = 1
+	max_upgrades = 3
+	fire_delay = 3
+	fire_sound = 'sound/weapons/guns/fire/smg_fire.ogg'
+	matter = list(MATERIAL_STEEL = 25, MATERIAL_PLASTEEL = 15, MATERIAL_DURASTEEL = 5)
+	init_firemodes = list(SEMI_AUTO_NODELAY, BURST_3_ROUND, BURST_5_ROUND, BURST_8_ROUND, FULL_AUTO_800)
+
```

Comment:
Weres its gun tags?

Reply:
Well I don't really know which tags should apply nor how much Possum will change it.